packages/customer-account-ui-extensions/src/components/PageBlock/PageBlock.ts

```diff
@@ -0,0 +1,30 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {CSSShorthand, Responsive, Spacing} from '../shared-components';
+
+type Background = 'transparent' | 'color1' | 'color2' | 'color3';
+
+export interface PageBlockProps {
+  /**
+   * Adjust the background.
+   */
+  background?: Background;
+  /**
+   * Adjust the padding.
+   *
+   * To shorten the code, it is possible to specify all the padding properties in one property.
+   *
+   *
+   * Examples:
+   *
+   * - `base` means blockStart, inlineEnd, blockEnd and inlineStart paddings are `base`
+   *
+   * - [`base`, `none`] means blockStart and blockEnd paddings are `base`, inlineStart and inlineEnd paddings are `none`
+   *
+   * - [`base`, `none`, `loose`, `tight`] means blockStart padding is `base`, inlineEnd padding is `none`, blockEnd padding is `loose` and  blockStart padding is `tight`
+   */
+  padding?: CSSShorthand<Spacing> | Responsive<CSSShorthand<Spacing>>;
+}
+
+export const PageBlock = createRemoteComponent<'PageBlock', PageBlockProps>(
```

Comment:
I understand its a component specific to Self Serve, but do you think 3rd party developers will need this in their extensions? Could this be wrapping extension points instead, so that developers choosing theses extension point are guaranteed to be consistent with the rest of the page?

Just curious to understand. Not a blocker.

Reply:
A full page app can potentially have multiple `PageBlocks` so in this case, it makes sense to expose it. But yeah, we'll also spend some more time sketching out the exact components we need/want to expose soon.